modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -96,7 +96,7 @@ spec:
 <3> Specify a label for the machine pool that you want to check.
 <4> Specify the machine set to track in `<cluster_name>-<label>-<zone>` format. For example, `prod-node-us-east-1a`.
 <5> Specify the timeout duration for the node condition. If the condition is met for the duration of the timeout, the machine will be remediated. Long timeouts can result in long periods of downtime for a workload on an unhealthy machine.
-<6> Specify the amount of unhealthy machines allowed in the targeted pool. This can be set as a percentage or an integer.
+<6> Specify the amount of machines allowed to be concurrently remediated in the targeted pool. This can be set as a percentage or an integer. maxUnhealthy prevents further remediation if the cluster is already partially unhealthy.
```

Comment:
(same comment as in other file)

Reply:
btw, it is very annoying to have this duplication. do you know why this text show up twice? Can the content be somehow merged?

Reply:
I agree, in general the less places to have to find the same thing to change it the better. I can look into this particular case to see if there's anything to be done about it.